Cargo.toml

```diff
@@ -29,12 +29,13 @@ crate-type = ["rlib"]
 [features]
 default = ["bundler", "grid", "nodejs"]
```

Comment:
I think we should probably add "sourcemap" to the default features. I guess the node bindings are getting it through "bundler" right now, though they kinda use it directly too even without the bundler.

Reply:
Ok great. Just force pushed a commit with sourcemap as an explicitly default feature